lib/src/query/query.dart

```diff
@@ -754,7 +754,7 @@ class Query<T> {
     try {
       final idArray = idArrayPtr.ref;
       return idArray.count == 0
-          ? List<int>.empty()
+          ? List<int>.filled(0, 0)
```

Comment:
Did this sneak in while trying to fix https://github.com/objectbox/objectbox-dart/issues/177?

Reply:
Nope, that's to restore compatibility with Dart 2.7